index.js

```diff
@@ -722,6 +722,16 @@ module.exports = {
     },
 
     /**
+     * Enable/Disable debug logs from Instabug SDK
+     * Default state: disabled
+     *
+     * @param isDebugEnabled whether debug logs should be printed or not into LogCat
+     */
+    setDebugEnabled: function(isDebugEnabled) {
+        Instabug.setDebugEnabled(isDebugEnabled);
```

Comment:
@salmatarzi  This line will throw exception  in case we don't have the same API for iOS 
Please add it for iOS or add an 'if' condition to make sure that we are on the proper OS